### DISCUSSION TITLE
How to show image like "logo.png"

### DISCUSSION THREAD

**Discussion by alnutile** [NativePHP/laravel#233] [Q&A] [answered]
Sorry if missed this in the docs but just needed help showing an image in say a tool bar.

```html
<img class="h-8 w-auto" src="/logo.png" alt="Test" />
```

and the file is in `public/logo.png`

Any link to the docs to help with this would be appreciated.

Thanks!

**Comment by simonhamp** [comment, marked as answer]
All files in your app get copied over and bundled into the build. When the app shell starts serving your Laravel app, everything should be accessible as it is normally. You shouldn't need to do anything special to those assets.
